migrations/env.py

```diff
@@ -2,6 +2,10 @@
 from alembic import context
 from sqlalchemy import engine_from_config, pool
 from logging.config import fileConfig
+import logging
+import re
+
+USE_TWOPHASE = False
```

Comment:
Good catch @oyiptong , I've also noticed that. We don't need the two phase commit here though. This option is from the Alembic template for the multi-db scenario, I didn't remove it mainly for the compatibility concern. I think it's OK here to keep all its default options. What do you think?